samples/analyze.v1p3beta1.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2019, Google, LLC
+ * Licensed under the Apache License, Version 2.0 (the `License`);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an `AS IS` BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+async function analyzeStreamingObject(path) {
+  const Video = require('@google-cloud/video-intelligence').v1p3beta1;
+  const fs = require('fs');
+  const util = require('util');
+  // Creates a client
+  const video = new Video.StreamingVideoIntelligenceServiceClient();
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // Reads a local video file and converts it to base64
+  const file = await util.promisify(fs.readFile)(path);
+  const inputContent = file.toString('base64');
+  const configRequest = {
+    videoConfig: {
+      streamingFeature: ['STREAMING_OBJECT_TRACKING'],
```

Comment:
According to the docs, this should be `feature: 'STREAMING_OBJECT_TRACKING',`

Reply:
@nirupa-kumar

Reply:
Fixed and updated with the latest styling format